requirements/core.txt

```diff
@@ -1,6 +1,6 @@
 python-dateutil ~= 2.8.2
 python-dotenv ~= 0.20.0
-progressbar2 ~= 3.55.0
+progressbar2 ~= 4.2.0
```

Comment:
I wanted to leave these dependencies entirely alone, but `progressbar2` needs to be consistent in `core.txt` and `docs.txt` to ensure the `tests.yml` workflow can run. The `sphinx` dependency in `dev.txt` was bumped for the same reason.